Tools/LambdaTestTool-v2/src/Amazon.Lambda.TestTool/Services/ApiGatewayRouteConfigService.cs

```diff
@@ -0,0 +1,98 @@
+using System.Collections;
+using System.Text.Json;
+using Amazon.Lambda.TestTool.Models;
+using Amazon.Lambda.TestTool.Services.IO;
+using Microsoft.AspNetCore.Routing.Template;
+
+namespace Amazon.Lambda.TestTool.Services;
+
+/// <inheritdoc cref="IApiGatewayRouteConfigService"/>
+public class ApiGatewayRouteConfigService : IApiGatewayRouteConfigService
+{
+    private readonly List<ApiGatewayRouteConfig> _routeConfigs = new();
+
+    /// <summary>
+    /// Constructs an instance of <see cref="ApiGatewayRouteConfigService"/>
+    /// which loads and parses environment variables that match a specific prefix.
+    /// </summary>
+    /// <param name="environmentManager">A service to access environment variables.</param>
+    public ApiGatewayRouteConfigService(
```

Comment:
Take in a ILogger and use debug logging through here. Getting the config wrong is likely going to be a place of problems and we will want some mechanism to asks users to enable logging so we can see what is going on.

Reply:
Done